app/controllers/users_controller.rb

```diff
@@ -489,93 +489,6 @@ def invited
     end
   end
 
-  def is_local_username
```

Comment:
Since all controller actions are considered public at Discourse, can you run a check against out internal logs to see if anyone is relying on this controller action as an API?

Reply:
I did not find any in the past two months. It is not a very interesting endpoint. 🤣